openhtmltopdf-core/src/main/java/com/openhtmltopdf/render/InlineLayoutBox.java

```diff
@@ -609,7 +609,7 @@ public Dimension positionRelative(CssContext cssCtx) {
     }
     
     // NOTE: Will be List of DisplayListItem when we delete the old renderer.
-    public void addAllChildren(List list, Layer layer) {
+    public void addAllChildren(List<Box> list, Layer layer) {
```

Comment:
I think this should be `? super Box` as it may be a list of `DisplayListItem`. `Box implements DisplayListItem`. I wasn't confident with wildcard generics when I wrote that comment (I still struggle).

Reply:
fixed :)